tests/integration/kubernetes/runtimeclass_workloads/confidential/unencrypted/Dockerfile

```diff
@@ -4,7 +4,8 @@
 
 # We know that using latest is error prone, we're taking the risk here.
 # hadolint ignore=DL3007
-FROM alpine:latest
+# alpine:3.20
+FROM alpine@sha256:b3119ef930faabb6b7b976780c0c7a9c1aa24d0c75e9179ac10e6bc9ac080d0d
```

Comment:
Could we use something like, `alpine:3.20` or `alpine:3.20.X`?

Reply:
I am worried to use a tag because it will move forward at some point of time, in which this error might happen again.

Reply:
I mean, will the `ssh-keygen` be updated for the old version (for example, 3.20.0) in the future?